src/apps/backend/modules/account/account_service.py

```diff
@@ -63,6 +68,21 @@ def get_account_by_username(*, username: str) -> Account:
     def get_account_by_username_and_password(*, params: AccountSearchParams) -> Account:
         return AccountReader.get_account_by_username_and_password(params=params)
 
+    @staticmethod
+    def get_notification_preferences(account_id: str) -> NotificationPreferences:
+        return AccountNotificationPreferenceReader.get_notification_preferences_by_account_id(account_id)
+
+    @staticmethod
+    def update_notification_preferences(params: UpdateNotificationPreferencesParams) -> NotificationPreferences:
```

Comment:
1. Function name is misleading - it should probably be `create_or_update_notification_preferences`. Update test name and other places based on this update.
2. Pull `account_id` from `UpdateNotificationPreferencesParams` and as an argument into `update_notification_preferences`